lib/link-cost.js

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Title : Test whether the link count in a page is too heavy or not.
+ *
+ *  This is done for SEO reasons, the behavior is based on asumption regarding cost of links for SEO.
```

Comment:
Title : Test whether there are too many links in a page

This is done for SEO reasons, based on the assumption that in the case of a lot of links not all of them would be followed and indexed.